common/python/tools/RDataFrame.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+# Author: Izaak Neutelings (November 2023)
+import os
+import ROOT; ROOT.PyConfig.IgnoreCommandLineOptions = True # to avoid conflict with argparse
+from ROOT import gROOT, gInterpreter, RDataFrame, RDF
+
+# REPRESENTATION: shorten for debugging
+RDataFrame.__repr__ = lambda o: "<%s at %s>"%(o.__class__.__name__,hex(id(o)))
+RDF.RInterface['ROOT::Detail::RDF::RJittedFilter,void'].__repr__ = lambda o: "<RInterface<RJittedFilter,void> at %s>"%(hex(id(o)))
+for temp in ['TH1D','TH2D','ULong64_t','double']: # templates
+  RDF.RResultPtr[temp].__repr__ = lambda o: "<%s at %s>"%(o.__class__.__name__,hex(id(o)))
+
+# PROGRESS BAR
+gInterpreter.Declare("""
```

Comment:
Right now this is compiled every time when running python. We should move this part to a macro that is compiled only once and loaded as a library. Like here: https://github.com/cms-tau-pog/TauFW/blob/90d4fbf217564f61c3033caece2911ade3f50085/Plotter/python/plot/MultiDraw.py#L23-L39